src/Documentation/Advanced-Concepts/Powershell-Client.md

```diff
@@ -137,7 +133,11 @@ PS> $grainId = [System.Guid]::Parse("A4CF7B5D-9606-446D-ACE9-C900AC6BA3AD")
 
 PS> $grain = Get-Grain -GrainType $grainInterfaceType -GuidKey $grainId
 
-PS> $message = $grain.SayHelloTo("Gutemberg").Result
+PS> $sayHelloTask = $grain.SayHelloTo("Gutemberg")
+
+PS> $sayHelloTask.Wait();
+
+PS> $message = $sayHelloTask.Result
```

Comment:
not sure I understand this change. Unless Wait() behaves differently in Powershell, it is doing a blocking call until the result is calculated, and then you just access the pre-calculated Result. Why not just call Result, which will block until the result is calculated too? Seems easier to have 1 line than 3.
Of course, if you want to parallelize, then that's another scenario, and you might want to store each task separately, but not necessarily use Wait() either, just Task.WaitAll.